kroxylicious-integration-tests/src/test/java/io/kroxylicious/proxy/filter/validation/JsonSchemaRecordValidationIT.java

```diff
@@ -108,7 +108,7 @@ class JsonSchemaRecordValidationIT extends RecordValidationBaseIT {
     public static void init() throws IOException {
         // An Apicurio Registry instance is required for this test to work, so we start one using a Generic Container
         DockerImageName dockerImageName = DockerImageName.parse("quay.io/apicurio/apicurio-registry-mem")
-                .withTag("2.5.11.Final");
+                .withTag("2.6.3.Final");
```

Comment:
Given the naming convention used for Docker versions matches the maven version, we could drive this from package metadata.